src/core/containers.py

```diff
@@ -34,3 +34,25 @@ def __init__(self, flow_name, task_name):
 
     def as_dict(self):
         return {"flow": self.flow, "task": self.task}
+
+
+class Populate(Container):
+    __slots__ = ["path", "validators", "value"]
+
+    def __init__(self, path="", validators="", value="", **kwargs):
+        super().__init__(**kwargs)
+        self.path = path
+        self.validators = validators
+        self.value = value
+
+    def as_dict(self):
+        retval = {}
+        if self.path:
+            retval.update(path=self.path)
+            if self.value:
+                raise Exception("'value' and 'path' attribute cannot be used together")
+        if self.value:
+            retval.update(value=self.value)
+        if self.validators:
```

Comment:
validators should be a required field as it how we decide when to populate the field

Reply:
Done